GANDLF/models/dynunet_wrapper.py

```diff
@@ -26,35 +62,26 @@ class dynunet_wrapper(ModelBase):
     def __init__(self, parameters: dict):
         super(dynunet_wrapper, self).__init__(parameters)
 
-        # checking for validation
-        assert (
-            "kernel_size" in parameters["model"]
-        ) == True, "\033[0;31m`kernel_size` key missing in parameters"
-        assert (
-            "strides" in parameters["model"]
-        ) == True, "\033[0;31m`strides` key missing in parameters"
-
-        # defining some defaults
-        # if not ("upsample_kernel_size" in parameters["model"]):
-        #     parameters["model"]["upsample_kernel_size"] = parameters["model"][
-        #         "strides"
-        #     ][1:]
-
+        patch_size = parameters.get("patch_size", None)
+        spacing = parameters.get(
+            "spacing", [1.0 for i in range(parameters["model"]["dimension"])]
+        )
+        kernel_size, strides = get_kernels_strides(patch_size, spacing)
+        parameters["model"]["kernel_size"] = parameters["model"].get(
+            "kernel_size", kernel_size
+        )
+        parameters["model"]["strides"] = parameters["model"].get("strides", strides)
```

Comment:
This ensures we do only run through compute when needed:
```suggestion
        parameters["model"]["kernel_size"] = parameters["model"].get(
            "kernel_size", None
        )
        parameters["model"]["strides"] = parameters["model"].get("strides", None)
        if (parameters["model"]["kernel_size"] is None) or (parameters["model"]["strides"] is None):
            kernel_size, strides = get_kernels_strides(patch_size, spacing)
```

Reply:
Done.